core-authentication/src/main/java/gg/solarmc/loader/authentication/HashingInstructions.java

```diff
@@ -19,21 +19,14 @@
 
 package gg.solarmc.loader.authentication;
 
-public final class HashingInstructions {
-
-	private final int iterations;
-	private final int memory;
-
-	HashingInstructions(int iterations, int memory) {
-		this.iterations = iterations;
-		this.memory = memory;
-	}
+public record HashingInstructions(int iterations, int memory) {
 
 	/**
 	 * Gets the amount of algorithm iterations
 	 *
 	 * @return the iteration
 	 */
+	@Override
```

Comment:
Why Override the method when the default method returns that same value which you are returning
probably remove it.

Reply:
I did that in order to provide javadocs. However, you are right, there is a better way to add javadocs to record components, so I'll do that.

Reply:
oh ok , you can do it like this too
```java
/**
* @param name The name of the record
* @param desc description of the record
*/
public record Record(String name, String desc){}
```
:P